source/Outfit.cpp

```diff
@@ -201,10 +216,21 @@ int Outfit::CanAdd(const Outfit &other, int count) const
 {
 	for(const auto &at : other.attributes)
 	{
+		// The minimum allowed value of most attributes is 0. Some attributes
+		// have special functionality when negative, though, and are therefore
+		// allowed to have values less than 0.
+		double minimum = 0.;
+		if(WHITELIST.count(at.first))
```

Comment:
Don't double-scan the map, by using count and then find.

Reply:
So instead 
```
auto it = WHITELIST.find(at.first);
if(it != WHITELIST.end()) {
	minimum = it->second;
	...
}
```
?